hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/PerformanceEvaluation.java

```diff
@@ -2308,7 +2316,7 @@ protected byte[] generateRow(final int i) {
     }
 
     @Override
-    boolean testRow(final int i, final long startTime) throws IOException {
+    protected boolean testRow(final int i, final long startTime) throws IOException {
```

Comment:
Why change this to protected?

Reply:
to override this in impl class, so that we can customise row format. If the custom test knows the dataset, it would be beneficial to repro any hotspotting scenarios like spin up multiple test client targeting the same dataset.

Reply:
But the methods in TestBase class are all package private? Even if you change this to protected, you can not inherit this class in other package because you can not call the constructor?

Reply:
>But the methods in TestBase class are all package private? Even if you change this to protected, you can not inherit this class in other package because you can not call the constructor?

Rest I am able to manage with by putting my impl classes in package with same name as PE, but without protected on SequentialWriteTest.testRow I am not able to override that method in my impl test class.

Reply:
Weirdly I am able to override PerformanceEvaluation.TableTest.testRow, but not PerformanceEvaluation.SequentialWriteTest.testRow when I put all impls in same package name.

Reply:
I was managing them by using the same package name, removing it